nfd/node-feature-rules-openshift.yaml

```diff
@@ -42,10 +42,16 @@ spec:
             vendor: {op: In, value: ["8086"]}
             device: {op: In, value: ["09c4"]}
             class: {op: In, value: ["1200"]}
-        - feature: kernel.loadedmodule
-          matchExpressions:
-            dfl_pci: {op: Exists}
-
+      matchAny:
```

Comment:
I prefer not to add the FPGA and IAA into the NFD if we haven't started to support them or will enable them in the coming release.

Reply:
I think we can keep IAA since it will be enabled in 1.1.0 release. I agree, we can remove FPGA and DLB as well. Can we keep IAA or remove it as well?

Reply:
I suggest to remove IAA, and only add what we will support.

Reply:
Makes sense! :)